addons/respawn/functions/fnc_adjustTimeLocal.sqf

```diff
@@ -17,7 +17,7 @@
 
 params ["_newTime"];
 
-if (playerRespawnTime isEqualTo -1) exitWith {};
+if (playerRespawnTime isEqualTo -1) exitWith {GVAR(oldTime) = _newTime;};
```

Comment:
"Never return assignment" ;-)
```suggestion
if (playerRespawnTime isEqualTo -1) exitWith {
    GVAR(oldTime) = _newTime;
    nil
};
```

Reply:
You never complained about it and now 2 times within 2 days. Has something changed? 😛

Reply:
No, it was always an issue. Maybe I'm looking more closely when I'm reviewing stuff for `Mods` than other repos :P